cmd/argo/commands/cron/create.go

```diff
@@ -66,13 +69,19 @@ func CreateCronWorkflows(filePaths []string, cliOpts *cliCreateOpts) {
 		os.Exit(1)
 	}
 
-	if cliOpts.schedule != "" {
-		for i := range cronWorkflows {
-			cronWorkflows[i].Spec.Schedule = cliOpts.schedule
+	for _, cronWf := range cronWorkflows {
+
+		if cliOpts.schedule != "" {
+			cronWf.Spec.Schedule = cliOpts.schedule
 		}
-	}
 
-	for _, cronWf := range cronWorkflows {
+		newWf := wfv1.Workflow{Spec: cronWf.Spec.WorkflowSpec}
+		err := util.ApplySubmitOpts(&newWf, submitOpts)
```

Comment:
should we not be applying these server-side?

Reply:
Submit applies this client side:

https://github.com/argoproj/argo/blob/3f293a4d647c6c10cf1bafc8d340453e87bd4351/cmd/argo/commands/submit.go#L211

I wanted to keep this simple (not modify the API).